### ISSUE TITLE
Training on my dataset collected with UR in pushT task,  Loss is too high

### ISSUE THREAD

**Issue by humphreyde** [moojink/openvla-oft#73]
Thank you for your impressive work! 
I'm currently attempting to reproduce your fine-tuning method on my  dataset. However, I've encountered an issue where the loss is too high:

![Image](https://github.com/user-attachments/assets/879c3d84-598e-4c4a-a9f1-88050ff1e769)

I collected 110 demos and split it to trainset(105) and valset(5), and translated the dataset to rlds  format, like:
```
steps:
	observation:
		image: 256x256x3
		wrist_image: 256x256x3
		state: 6 dim joint_states
	action: 6 dim end pose
	discount: default
	reward: default
	is_first: default
	is_last: default
	is_terminal: default
	language_instruction: "put T-block into area"
```
since I didn't use gripper, action dim was 6.
visualize it like:

config.py:
```python
"ur_put_t_into_area_110_demos": {
"image_obs_keys": {"primary": "image", "secondary": None, "wrist": "wrist_image"},
"depth_obs_keys": {"primary": None, "secondary": None, "wrist": None},
"state_obs_keys": ["state"],
"state_encoding": StateEncoding.POS_EULER,
"action_encoding": ActionEncoding.EEF_POS,
},
```
constants.py:
```python
UR_CONSTANTS = {
"NUM_ACTIONS_CHUNK": 10,
"ACTION_DIM": 6,
"PROPRIO_DIM": 6,
"ACTION_PROPRIO_NORMALIZATION_TYPE": NormalizationType.BOUNDS,
}
```

 Any suggestions or guidance would be greatly appreciated!
@moojink

**Comment by agdfgsdgfsdukyukjhjsdffsdfsdfds** [comment]
Did you solve your problem? What constants did you choose in both dicts (especially in `UR_CONSTANTS`)?

**Comment by humphreyde** [comment]
> Did you solve your problem? What constants did you choose in both dicts (especially in `UR_CONSTANTS`)? 

yes，just as the code shows.

**Comment by moojink** [comment]
Hi @humphreyde, how many GPUs are you using and what is the global batch size? Also, are the training demonstrations collected with a consistent strategy or are they very multimodal?

Given the large oscillations in the training curves, it's possible that the global batch size is too low. Or the dataset is just strangely difficult to fit.
